frontends/web/src/components/layout/header.module.css

```diff
@@ -54,7 +58,8 @@
 
 .title {
     white-space: nowrap;
-    flex: 1;
+    flex-grow: 1;
+    margin-bottom: var(--space-quarter);
 }
```

Comment:
could you add some padding-right so that there is always a little space between the title and the icon?

![Screen Shot 2023-11-07 at 15 33 59](https://github.com/digitalbitbox/bitbox-wallet-app/assets/546900/2928f3c0-cefc-4dc3-862e-79e977d16d3b)